exporter/sentryexporter/config_test.go

```diff
@@ -45,6 +45,6 @@ func TestLoadConfig(t *testing.T) {
 			NameVal: "sentry/2",
 			TypeVal: "sentry",
 		},
-		DSN: "https://key@host:port/1",
+		DSN: "https://publicKey:secretKey@host/path/id",
```

Comment:
Can we please avoid `public:private`? That's obsolete stuff.

```suggestion
		DSN: "https://key@host/path/id",
```